src/features/auth/lib/authMachine.ts

```diff
@@ -197,14 +197,14 @@ export const authMachine = createMachine<
             target: "connectingToMetamask",
             cond: (context) => context.wallet === "METAMASK",
           },
-          {
-            target: "connectingToWalletConnect",
-            cond: (context) => context.wallet === "WALLET_CONNECT",
-          },
           {
             target: "connectingToSequence",
             cond: (context) => context.wallet === "SEQUENCE",
           },
+          {
+            target: "connectingToWalletConnect",
+            cond: (context) => !!context.wallet,
```

Comment:
Can you explain this change?

Reply:
@spencerdezartsmith Yes that is right - Wallet Connect will provide whichever wallet is chosen 😄